File: BEE 1150.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    int x, z;
    cin >> x;
    cin >> z;
    while(z <= x){ cin >> z;}
    int sum = 0;
    int cnt = 1;
    for (int i = x; i < z; i++){
        sum += i;
        if (sum < z){
            cnt += 1;
        }
    }
    cout << cnt << endl;
    return 0;
}

File: BEE 1150.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var x = parseInt(lines.shift());
var z = parseInt(lines.shift());
while(z <= x){ z = parseInt(lines.shift());}
var sum = 0;
var cnt = 1;
for (var i = x; i < z; i++){
    sum += i;
    if (sum < z){
        cnt += 1;
    }
}
console.log(cnt);